scripts/build.py

```diff
@@ -1102,12 +1105,15 @@ def build():
     parser = optparse.OptionParser()
     parser.add_option(
         '--prod_env', action='store_true', default=False, dest='prod_mode')
+    parser.add_option(
+        '--prod_minify_only', action='store_true', default=False,
+        dest='minify_only')
     options = parser.parse_args()[0]
     # Regenerate /third_party/generated from scratch.
     safe_delete_directory_tree(THIRD_PARTY_GENERATED_DEV_DIR)
     build_third_party_libs(THIRD_PARTY_GENERATED_DEV_DIR)
     if options.prod_mode:
-        generate_build_directory()
+        generate_build_directory(options.minify_only)
```

Comment:
I feel like it is cleaner to do a 2nd option here instead of tacking an argument on top of `generate_build_directory`
```suggestion
        generate_build_directory()
        if options.minify_third_party_libs_only:
            minify_third_party_libs(THIRD_PARTY_GENERATED_DEV_DIR)
```
What do you think @sshou14?

Reply:
Sure! I agree